app-frontend/src/app/components/scenes/sceneImportModal/sceneImportModal.html

```diff
@@ -194,13 +194,20 @@ <h3 class="no-margin">Local File Import</h3>
           </button>
         </div>
         <div class="list-group">
-          <div class="list-group-item" ng-repeat="file in $ctrl.selectedFiles">
+          <div class="list-group-item" ng-repeat="fileDS in $ctrl.selectedFileDatasets">
             <div class="list-group-overflow">
+              <span class="icon-warning"
+                ng-if="fileDS.bandCount !== $ctrl.datasource.bands.length"
+                tooltips
+                tooltip-template="This file has {{fileDS.bandCount}} band(s) but the Datasource you chose specifies {{$ctrl.datasource.bands.length}}."
```

Comment:
If the datasource doesn't have any bands defined, it shows no number instead of 0

Reply:
I think I'd add the `color-warning` class to make it pop out a little as well

Reply:
Good points. I added the color-warning class, and added a second tool tip to change the message if the datasource doesn't define any bands. Now it looks like this:
<img width="872" alt="screen shot 2018-03-06 at 9 35 05 am" src="https://user-images.githubusercontent.com/447977/37038064-bc6ea288-2121-11e8-9a66-77c661812bf9.png">